modules/launcher/src/main/resources/dbscripts/oracle-mb.sql

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+-- WSO2 Message Broker Oracle Database schema --
+
+-- Start of Message Store Tables --
+/
+CREATE TABLE MB_EXCHANGE (
```

Comment:
Can we add the "if not exists" check?

Reply:
There doesn't seem to be a straightforward/simple way to do an "if exists" check for table creation with Oracle. We would have to drop the tables if they exist separately (keeping this script simple), prior to creating the tables?

Reply:
👍 Let's keep the script as it is then. Shall we mention that we need to drop old tables in the doc?

Reply:
Created #233 to track introducing relevant sections in the database setup guide. Will include dropping old tables for Oracle.